source/unicodeScriptPrep.py

```diff
@@ -0,0 +1,48 @@
+#A part of NonVisual Desktop Access (NVDA)
+#Copyright (C) 2015 Dinesh Kaushal, NV Access Limited
+#This file is covered by the GNU General Public License.
+#See the file COPYING for more details.
+
+"""unicodeScriptPrep module is used to build unicodeScriptData.py file. This module obtains scripts.txt from unicode.org and builds an array of unicode ranges. 
+After  the array is created, it is written to the unicodeScriptData.py module.
+To generate the unicodeScriptData.py, just run this script with python."""
+
+import urllib2, re, textwrap
+
+def generateUnicodeScriptData():
+	"""build unicodeScriptData.py from scripts.txt"""
+	unicodeRange= []
+	url = 'http://www.unicode.org/Public/UNIDATA/Scripts.txt'
+	scriptDataFile = urllib2.urlopen(url)
+	for line in scriptDataFile:
+		p = re.findall(r'([0-9A-F]+)(?:\.\.([0-9A-F]+))?\W+(\w+)\s*#\s*(\w+)', line)
```

Comment:
Please provide an example (in a comment) of the data that this should work on, including edge cases that had to be worked around.